src/main/java/com/greenfox/lvlup/model/MockingElements.java

```diff
@@ -0,0 +1,88 @@
+package com.greenfox.lvlup.model;
+
+import com.greenfox.lvlup.controller.PitchDto;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class MockingElements {
+
+  private String validToken = "token123";
+
+  private PitchDto validPitchDto = new PitchDto("English speaker",
```

Comment:
Creating objects like this makes our code hardly readable, plus it will fill it with magic strings, try to outsource these strings into variables and create the objects from that, it will look much readable

Reply:
Done.